x/crosschain/keeper/keeper_in_tx_tracker.go

```diff
@@ -0,0 +1,101 @@
+package keeper
```

Comment:
I think we can remove keeper_ prefix in file name since it is already in keeper package

Reply:
I think that makes sense. However, this PR follows the naming used by the cross-chain module so far. We can take the naming refactor on as a separate issue

Reply:
I actually created a PR for this https://github.com/zeta-chain/node/pull/963
But it's hard to manage since you get conflicts every time the file is changed. That's why I think it's easier to already rename the file on the flow.
There are already files not prefixed with keeper also